examples/auth/basic/auth.go

```diff
@@ -127,14 +152,19 @@ func callInitialRequest(mockedWallet wallet.WalletInterface) *transport.AuthMess
 	return &result
 }
 
-func callPingEndpoint(mockedWallet wallet.WalletInterface, response *transport.AuthMessage) {
-	url := "http://localhost:8080/ping"
+// Makes an authenticated request to the ping endpoint
```

Comment:
Even for private function I would stick to documentation format where a name of described entity is as the initial word. 

```suggestion
// callPingEndpoint makes an authenticated request to the ping endpoint
```

Reply:
fixed comments for "non-exported" members